intelmq/tests/bots/collectors/github_api/test_collector.py

```diff
@@ -95,7 +92,6 @@ def print_requests_get_parameters(url, *args, **kwargs):
         main_mock = MagicMock(content=EXAMPLE_CONTENT_STR)
         return main_mock
 
-
```

Comment:
As [you are keen on codestyle and "Quality of Life"](https://github.com/certtools/intelmq/pull/2211): Before a class statement there should be two empty lines ;)